main.py

```diff
@@ -8,3 +11,6 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+t2 = time.clock()
+print((t2-t1)/60)
```

Comment:
Finir le script avec une newline :
```suggestion
print((t2-t1)/60)

```